### PR TITLE
Add model development notebook

### PR DESCRIPTION
**Pull request by SSaishruthi**
Made updates to code and documentation

### CODE REVIEW

File: notebooks/Part 2 - Model Development.ipynb [hunk 1]
```diff
@@ -1 +1 @@
... AI Technologies](http://codait.org).\n\nCopyright \u00a9 2020 IBM. This notebook and its source code are released under the terms of the MIT License."}, {"metadata": {}, "cell_type": "markdown", "source": "<div style=\"background:#F5F7FA; height:110px; padding: 2em; font-size:14px;\">\n<span style=\"font-size:18px;color:#152935;\">Love this notebook? </span>\n<span style=\"font-size:15px;color:#152935;float:right;margin-right:40px;\">Don't have an account yet?</span><br>\n<span style=\"color:#5A6872;\">Share it with your colleagues and help them discover the power of Watson Studio!</span>\n<span style=\"border: 1px solid #3d70b2;padding:8px;float:right;margin-right:40px; color:#3d70b2;\"><a href=\"https://ibm.co/wsnotebooks\" target=\"_blank\" style=\"color: #3d70b2;text-decoration: none;\">Sign Up</a></span><br>\n</div"}], "metadata": {"kernelspec": {"name": "python3", "display_name": "Python 3.7", "language": "python"}, "language_info": {"name": "python", "version": "3.7.9", "mimetype": "text/x-python", "codemirror_mode": {"name": "ipython", "version": 3}, "pygments_lexer": "ipython3", "nbconvert_exporter": "python", "file_extension": ".py"}}, "nbformat": 4, "nbformat_minor": 4}
```
Comment (tlzhu19) [inline notebooks/Part 2 - Model Development.ipynb:1]: <p>You can delete line 17 to end (<code>from sklearn.cluster import KMeans</code>), these imports were used in the clustering part.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/watson-studio-gallery-debater-sentiment-composition-lexicons-project/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (tlzhu19) [inline notebooks/Part 2 - Model Development.ipynb:1]: <p>You can add to 1.1 Load Data:</p><p>We will load the cleaned files 'unigrams.csv' and 'bigrams.csv' from Part 1 as well as the rule files 'ADJECTIVES.xlsx' and 'SEMANTIC_CLASSES.xlsx' from the original dataset .</p><p> </p><p> </p><p>Then,</p><p>In 1.2 and 1.3, instead of 'LEXICON_UG.txt'&nbsp;say 'unigrams.csv' and instead of 'LEXICON_BG.txt' use 'bigrams.csv'</p>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/watson-studio-gallery-debater-sentiment-composition-lexicons-project/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (tlzhu19) [inline notebooks/Part 2 - Model Development.ipynb:1]: <p>Add explanation:</p><p> </p><p>In this first implementation (calculate_unigram_sentiment()), we use the unigrams dataset, which contains the sentiment of various unigrams. Given an input sentence, we tokenize the sentence (break it up as a list of words) then match each word in the sentence to the words in the unigram dataset and find its sentiment value. If the word is not found in the dataset, it is skipped.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/watson-studio-gallery-debater-sentiment-composition-lexicons-project/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (tlzhu19) [inline notebooks/Part 2 - Model Development.ipynb:1]: <p>add sentence above this cell: </p><p>Below is an example using this simple implementation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/watson-studio-gallery-debater-sentiment-composition-lexicons-project/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (tlzhu19) [inline notebooks/Part 2 - Model Development.ipynb:1]: <p>Add explanation:</p><p>This second implementation (calculate_bigram_sentiment()) is similar to the first unigrams implementation however it uses bigrams instead. Given an input sentence, we tokenize the sentence into bigrams (e.g. "the good dog" becomes something like [("the", "good"), ("good", "dog")]) then match each bigram in the sentence to the bigrams in the dataset and find its sentiment value. If a bigram is not found in the dataset, it is skipped.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/watson-studio-gallery-debater-sentiment-composition-lexicons-project/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (tlzhu19) [inline notebooks/Part 2 - Model Development.ipynb:1]: <p>Add:</p><p> </p><p>Below is an example using the bigram implementation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/watson-studio-gallery-debater-sentiment-composition-lexicons-project/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (tlzhu19) [inline notebooks/Part 2 - Model Development.ipynb:1]: <p>sorry I had some typos originally so please change:</p><ul><li>change "ti" to "to"</li><li>in the line 1., say section 1.3 rather than 1.2</li><li>in the line 2., say section 1.4 rather than 1.3</li><li>in the line 3., say section 1.2 calculate_unigram_sentiment()</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/watson-studio-gallery-debater-sentiment-composition-lexicons-project/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (tlzhu19) [inline notebooks/Part 2 - Model Development.ipynb:1]: <p>modify to say:</p><p> </p><p>Now we add a method to group the entire comment's overall sentiment rather than just its individual sentences. Within each comment, we also label the positive/negative sentences.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/watson-studio-gallery-debater-sentiment-composition-lexicons-project/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (tlzhu19) [inline notebooks/Part 2 - Model Development.ipynb:1]: <p>In the below example we visualize in a more human readable way the sentiment of each comment overall as well as each sentence in each comment.</p><p> </p><p>In this example, two comments are overall negative, two comments overall are positive, and one is neutral.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/watson-studio-gallery-debater-sentiment-composition-lexicons-project/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (tlzhu19) [inline notebooks/Part 2 - Model Development.ipynb:1]: <p>not sure why there's an extra "&lt;/div" showing, delete that / update the cell </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/watson-studio-gallery-debater-sentiment-composition-lexicons-project/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Comment (SSaishruthi) [reply notebooks/Part 2 - Model Development.ipynb:1]: <p>Updated with respect to extracting data path.</p>

Comment (SSaishruthi) [reply notebooks/Part 2 - Model Development.ipynb:1]: <p>done</p>

Comment (SSaishruthi) [reply notebooks/Part 2 - Model Development.ipynb:1]: <p>done</p>

Comment (SSaishruthi) [reply notebooks/Part 2 - Model Development.ipynb:1]: <p>done</p>

Comment (SSaishruthi) [reply notebooks/Part 2 - Model Development.ipynb:1]: <p>done</p>

Comment (SSaishruthi) [reply notebooks/Part 2 - Model Development.ipynb:1]: <p>done</p>

Comment (SSaishruthi) [reply notebooks/Part 2 - Model Development.ipynb:1]: <p>done</p>

Comment (SSaishruthi) [reply notebooks/Part 2 - Model Development.ipynb:1]: <p>updated section header too from 2.6 -&gt; 1.6</p><p> </p><p>Done</p>

Comment (SSaishruthi) [reply notebooks/Part 2 - Model Development.ipynb:1]: <p>Need to close that div. Done</p>

**Comment by SSaishruthi** [comment]
@tlzhu19 removed first hidden cell that had project token

File: notebooks/Part 2 - Model Development.ipynb (hunk 1 above)
Comment (tlzhu19) [inline notebooks/Part 2 - Model Development.ipynb:1]: <p>sorry didn't catch this earlier, under 1.5 you should change 1.2 to 1.3, 1.3 to 1.4 and 1.1 to 1.2 calculate_unigram_sentiment()</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/watson-studio-gallery-debater-sentiment-composition-lexicons-project/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (SSaishruthi) [reply notebooks/Part 2 - Model Development.ipynb:1]: Yea, made those changes.

**Review by tlzhu19 — APPROVED** [review]
